packages/create-invoice-form/README.md

```diff
@@ -24,6 +24,12 @@ npm install @requestnetwork/create-invoice-form
 
 You can directly pass props into the create-invoice-form web component without needing to create references or use workarounds.
 
+##### Important Note on Currencies Prop
+
+- The currencies prop is now optional.
+- If you include the currencies prop and follow the proper format, it will override the default currencies.
+- To use the default currencies list, simply omit the currencies prop.
+
```

Comment:
Recommend moving this down to be just above the currencies examples.

Please also adjust the header levels so that you're not skipping levels.